src/Util/MIMETypeUtil.h

```diff
@@ -52,4 +53,8 @@ extern NSString *const OWSMimeTypeOversizeTextMessage;
 + (NSSet<NSString *> *)supportedImageUTITypes;
 + (NSSet<NSString *> *)supportedAnimatedImageUTITypes;
 
++ (NSString *)utiTypeForMIMEType:(NSString *)mimeType;
++ (NSString *)fileExtensionForUTIType:(NSString *)utiType;
++ (NSString *)fileExtensionForMIMEType:(NSString *)mimeType;
```

Comment:
These functions are used in the new "arbitrary attachments" changes to Signal-iOS.